components/account/AccountFollowButton.vue

```diff
@@ -13,6 +13,11 @@ const { t } = useI18n()
 const isSelf = useSelfAccount(() => account)
 const enable = computed(() => !isSelf.value && currentUser.value)
 const relationship = computed(() => props.relationship || useRelationship(account).value)
+const isLoading = ref(false)
+
+watchEffect(() => {
+  isLoading.value = relationship.value === undefined
+})
```

Comment:
Could this be rewritten as:
```suggestion
const isLoading = computed(() => relationship.value === undefined)
```
?

Reply:
That's correct 👍🏻